content/docs/gke/gcp-e2e.md

```diff
@@ -945,6 +930,17 @@ Follow these steps to access the web UI in your web browser.
 1. Click **Connect**. The system finds the server in your cluster and displays
    the classification results.
 
+As an alternative to the external IP address, you can run the following command 
+to access the UI via `kubectl port-forward`:
+
+```
+kubectl port-forward -n ${DEPLOYMENT_NAME}  `kubectl get pods --all-namespaces \
```

Comment:
This isn't exactly right. -n specifies the namespace. In this case, DEPLOYMENT_NAME==namespace==kubeflow, but if the DEPLOYMENT_NAME is something else, you should still enter "-n kubeflow"

Reply:
Ah thanks.